### PR TITLE
Drop support for installing a plugin with a specific version

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Since https://github.com/mattermost/mattermost-server/pull/19613, the Marketplace install endpoint always installs the latest version of a given plugin.

This PR updated `mmctl` to take that into account and drops the support for passing a version as part of the command. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-42721

### CODE REVIEW

File: commands/plugin_marketplace.go [hunk 1]
```diff
@@ -21,15 +19,12 @@ var PluginMarketplaceCmd = &cobra.Command{
...
 var PluginMarketplaceInstallCmd = &cobra.Command{
-	Use:   "install <id> [version]",
+	Use:   "install <id>",
 	Short: "Install a plugin from the marketplace",
 	Long:  "Installs a plugin listed in the marketplace server",
-	Example: `  # you can specify with both the plugin id and its version
-  $ mmctl plugin marketplace install jitsi 2.0.0
-
-  # if you don't specify the version, the latest one will be installed
+	Example: `  # you can specify with both the plugin id
```
Comment (isacikgoz) [inline commands/plugin_marketplace.go:25]:
```suggestion
	Example: `  # you can specify the plugin id
```

File: docs/mmctl_plugin_marketplace_install.rst [hunk 2]
```diff
@@ -13,17 +13,14 @@ Installs a plugin listed in the marketplace server
...
 Examples
 ~~~~~~~~
 
 ::
 
-    # you can specify with both the plugin id and its version
-    $ mmctl plugin marketplace install jitsi 2.0.0
-
-    # if you don't specify the version, the latest one will be installed
+    # you can specify with both the plugin id
```
Comment (isacikgoz) [inline docs/mmctl_plugin_marketplace_install.rst:23]: You may need to regenerate the docs as well. 
```suggestion
    # you can specify the plugin id
```

**Review by isacikgoz — APPROVED** [review]
Looks good, just caught a couple of typos.

File: commands/plugin_marketplace.go (hunk 1 above)
Comment (hanzei) [reply commands/plugin_marketplace.go]: Good catch, updated it

**Comment by hanzei** [comment]
@isacikgoz Please merge at your convenience

**Pull request merged**